packages/core/src/Stores/client-store.js

```diff
@@ -2023,15 +2023,9 @@ export default class ClientStore extends BaseStore {
         }
 
         const is_client_logging_in = login_new_user ? login_new_user.token1 : obj_params.token1;
-
         if (is_client_logging_in) {
             const is_pre_appstore = window.localStorage.getItem('is_pre_appstore');
-            const redirect_url = sessionStorage.getItem('redirect_url');
-            if (
-                is_pre_appstore === 'true' &&
-                redirect_url?.endsWith('/') &&
-                (isTestLink() || isProduction() || isLocal() || isStaging())
-            ) {
+            if (is_pre_appstore === 'true' && (isTestLink() || isProduction() || isLocal() || isStaging())) {
```

Comment:
Decided to remove the `redirect_url?.endsWith('/')`, because I believe we should redirect to trader's hub dashboard if `pre_app_store` is `enabled`. I did a sanity check and everything seemed fine, I'll put it back if it's really needed.